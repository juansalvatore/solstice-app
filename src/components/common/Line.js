import React from 'react'
import styled from 'styled-components'

const Line = () => <StyledLine />

export default Line

const StyledLine = styled.span`
  position: absolute;
  height: 1px;
  background-color: rgba(0, 0, 0, 0.1);
  left: 50%;
  transform: translateX(-50%);
  width: calc(100% - 40px);
`
