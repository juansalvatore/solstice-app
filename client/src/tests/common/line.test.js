import React from 'react'
import Line from '../../components/common/Line'
import { shallow } from 'enzyme'
import renderer from 'react-test-renderer'

describe('Line component', () => {
  it('matches the snapshot', () => {
    const tree = renderer.create(<Line />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
