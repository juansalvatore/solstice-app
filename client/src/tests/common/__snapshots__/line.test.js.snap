// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Line component matches the snapshot 1`] = `
<span
  className="sc-bdVaJa ekyfYm"
/>
`;
